working_agreement.md

```diff
@@ -28,14 +29,14 @@
 1. Building a diverse environment
     - Please be open to others perspectives, engage with them, and be willing to revise your own
```

Comment:
Maybe add a line within values about "We will be aware and acknowledge our positionalities and privileges that we hold"